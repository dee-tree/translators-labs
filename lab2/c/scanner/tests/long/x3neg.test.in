L#-12465656