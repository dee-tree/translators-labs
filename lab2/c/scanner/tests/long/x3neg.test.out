token	\256 	yytext -12465656	yyleng 9	yylval -12465656
token	\0 	yytext 	yyleng 1	yylval -12465656
